client/team-info.js

```diff
@@ -1,25 +1,27 @@
-    { name: 'Karissa McKelvey', github: 'karissa', description: 'Karissa is a board member at Code for Science & Society and worked on the Dat Project since 2014. She is currently an open source developer at Digital Democracy. Former <a href="http://scholar.google.com/citations?user=RM2tB8EAAAAJ&hl=en">academic</a> experienced in building interactive data visualization and collaboration tools.' },
+    { name: 'Karissa McKelvey', github: 'karissa', description: 'Karissa is a board member at Code for Science & Society and worked on Dat since 2014. She is currently an open source developer at Digital Democracy. Former <a href="http://scholar.google.com/citations?user=RM2tB8EAAAAJ&hl=en">academic</a> experienced in building interactive data visualization and collaboration tools.' },
     { name: 'Danielle Robinson, PhD', github: 'daniellecrobinson', description: 'Danielle Robinson is Co-Executive Director of Code for Science & Society where she works support and advise people who create open source technology in the public interest. In 2016, she completed a PhD in neuroscience at Oregon Health & Science University. Then as a Mozilla Fellow for Science she ran Working Open Workshops, explored decentralized approaches to data preservation, and advocated for open access to research.' },
     { name: 'Joe Hand', github: 'joehand', description: 'Joe Hand is Co-Executive Director at Code for Science & Society and a core developer on Dat Project with a focus on developing and managing data-focused programs for researchers and community-driven organizations. Previously, Joe managed a global project at the Santa Fe Institute to transform data collection practices of an international NGO, operating in slums across 30 countries in the Global South' }
   ],
+  'other': [
+    {name: 'Georgiy Shibaev', github: 'RangerMauve', description: 'Based in Ottowa, Canada. Open source software engineer focused on developer experience, the command line tool, and documentation.'},
+    {name: 'Andrew Osheroff', github: 'andrewosh', description: 'Based in Copenhagen, improving performance and APIs in Hyperdrive.'}
+   ],
   'wg': [
-    { name: 'Mathias Buus', github: 'mafintosh', description: 'Beaker Browser' },
-    { name: 'Tara Vancil', github: 'taravancil', description: 'Beaker Browser' },
+    { name: 'Mathias Buus', github: 'mafintosh', description: 'Hyperdivision' },
     { name: 'Paul Frazee', github: 'pfrazee', description: 'Beaker Browser' },
-    { name: 'Karissa McKelvey', github: 'karissa', description: 'Code for Science & Society, Digital Democracy' },
+    { name: 'Karissa McKelvey', github: 'karissa', description: 'Dat Project, Digital Democracy' },
```

Comment:
Right now we do not show these, do you think we should also include these descriptions on the site?

Reply:
Might be a good idea. Also, @pvh had a nice idea that we could limit the number of folks from the same organization that are able to participate in the wg (e.g., 2 people per org) which is what Postgres did in the early days, and seemed to work for them to avoid takeovers by a single org.

Reply:
definitely for a larger conversation but i think highlights the importance of showing the affiliated organizations here.

Reply:
Okay, added to master (sorry thought this was already merged for some reason): https://github.com/datproject/datproject.org/commit/ce7c8ec02c10ad612d4e9725ef06271f9ed8b78a